src/constants/storageKeys.js

```diff
@@ -1,4 +1,5 @@
-export const ACCESS_TOKEN_KEY = 'access_token';

```

Comment:
Why are these lines considered changed? I hope it's not because of line endings...